content/concept-amendments.md

```diff
@@ -0,0 +1,164 @@
+
+In the flag ledger itself, there is nothing unusual. However, during that time, the servers look at the votes of the validators they trust, and decide whether to insert an [`EnableAmendment` pseudo-transaction](reference-transaction-format.html#enableamendment) into the following ledger. The flags of an EnableAmendment pseudo-transaction indicate what the server thinks happened:
+
+* The `tfGotMajority` flag means that support for the amendment has increased to at least 80% of trusted validators.
+* The `tfLostMajority` flag means that support for the amendment has decreased to less than 80% of trusted validators.
+* An EnableAmendment pseudo-transaction with no flags means that support for the amendment has been enabled. (The change in transaction processing applies to every ledger after this one.)
+
+A server only inserts the pseudo-transaction to enable an amendment if all of the following conditions are met:
+
+* The amendment has not already been enabled.
+* A previous ledger includes an EnableAmendment pseudo-transaction for this amendment with the `tfGotMajority` flag enabled.
+* The previous ledger in question is an ancestor of the current ledger.
+* The previous ledger in question has a close time that is at least **two weeks** before the close time of the latest flag ledger.
+* There are no EnableAmendment pseudo-transactions for this amendment with the `tfLostMajority` flag enabled in the consensus ledgers between the `tfGotMajority` pseudo-transaction and the current ledger.
+
+It is theoretically possible (but extremely unlikely) that a `tfLostMajority` EnableAmendment pseudo-transaction could be included in the same ledger as the pseudo-transaction to enable an amendment. In this case, the pseudo-transaction with the `tfLostMajority` pseudo-transaction has no effect.
+
+## Amendment Voting ##
+
+Operators of `rippled` validators can choose which amendments to support or reject using the [`feature` command](reference-rippled.html#feature). This decides which amendments the validator votes for in the [amendment process](#amendment-process). By default, `rippled` votes in favor of every amendment it knows about.
+
+The operator of a `rippled` validator can "veto" an amendment. In this case, that validator never sends a vote in favor of the amendment. If enough servers veto an amendment, that prevents it from reaching consistent 80% support, so the amendment does not apply.
+
+As with all aspects of the consensus process, amendment votes are only taken into account by servers that trust the validators sending those votes. Currently, Ripple recommends only trusting the 5 default validators that Ripple (the company) operates. For now, trusting only those validators is sufficient to coordinate with Ripple on releasing new features.
```

Comment:
Would it be worth linking to the validator registry here? 

Also, does the last instance of "Ripple" in the last sentence refer to the company or the network? (Sorry about this one.)